build.gradle

```diff
@@ -1,113 +1,43 @@
-buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath "com.netflix.nebula:nebula-publishing-plugin:17.3.0"
```

Comment:
These should also be included, they help with testing locally (mavenLocal) or else how do you test it?